src/cow-react/modules/limitOrders/state/limitOrdersWarningsAtom.ts

```diff
@@ -0,0 +1,19 @@
+import { atom } from 'jotai'
+
+interface LimitOrdersWarnings {
+  isRateImpactAccepted: boolean
+  isPriceImpactAccepted: boolean
```

Comment:
I'm just wondering. If the user:
- Accepts the impact
- Then, something changes (maybe the USD estimation), or maybe the user changes the price
- Do they need to accept the impact again (if it changes for worse)?

Reply:
https://github.com/cowprotocol/cowswap/pull/1414/files#diff-a9c6e86ffae068b345c0abd2a8fe5087e58c9809342aee61a8c758e8292649cfR49-R59

We reset these flags if anything changes